test/src/test/java/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitorTest.java

```diff
@@ -371,7 +371,7 @@ private void checkUserWithLegacyTokenListHasSizeOf(
     
     private void simulateUseOfLegacyToken(User user) throws Exception {
         JenkinsRule.WebClient wc = j.createWebClient();
-        wc.withBasicApiToken(user);
+        wc.withBasicCredentials(user.getId(), user.getProperty(ApiTokenProperty.class).getApiToken());
```

Comment:
This test specifically needs to use the old implementation, so inlining that here.